sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -85,7 +113,13 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         uniqueFileName: true,
         fileContent: true,
       },
-      where: { uniqueFileName, virusScanStatus: VirusScanStatus.InProgress },
+      where: {
+        uniqueFileName,
+        virusScanStatus: In([
+          VirusScanStatus.InProgress,
```

Comment:
Optional and may not be required at this time: Can we make this method generic to be able to retrieve the student files with any virus scan status using their unique file name?

Reply:
This is a private method and this service is already not following the standards.
Ideally, the DB methods would leave a separate service from the service processing the virus scanning.
Please refrain from investing time in this right now. If I am missing something please let me know.

Reply:
Yes, I remember now. You explained it to me in one of my earlier PRs. 👍